dev-libs/liboil/liboil-0.3.17-r4.ebuild

```diff
@@ -24,7 +24,8 @@ BDEPEND="
 PATCHES=( "${FILESDIR}"/${P}-amd64-cpuid.patch )
 
 src_prepare() {
-	has x32 $(get_all_abis) && PATCHES+=( "${FILESDIR}"/${PN}-0.3.17-x32.patch )
+	[[ ${CHOST} == *x32 ]] && PATCHES+=( "${FILESDIR}"/${PN}-0.3.17-x32.patch )
```

Comment:
Why are we even applying this conditionally?

Reply:
I wonder the same, although I lack the experience with x32 to judge whether it should be applied unconditionally. I don't know if the change is sufficiently portable with other compilers and such.